src/renderer/components/dex/TradeHistory.vue

```diff
@@ -32,10 +33,17 @@ export default {
     storeUnwatch();
   },
 
+  computed: {
+    trades() {
+      return this.$store.state.tradeHistory && this.$store.state.tradeHistory.trades
+        && this.$store.state.tradeHistory.trades.length ? this.$store.state.tradeHistory.trades : [];
+    },
+  },
+
   mounted() {
     this.loadTrades();
     loadTradesIntervalId = setInterval(() => {
-      this.loadTrades();
+      this.loadTrades(true);
```

Comment:
Another great place to abstract to make it clear what `true` means. 

`loadTradesSilently`